app/utils/local_drive_identity.py

```
@@ -7,18 +7,17 @@
-from typing import Dict, Optional
 
 from app.models import ColdStorageBackendType, ColdStorageLocation
 
 
-def _diskutil_info(path: Path) -> Dict:
+def _diskutil_info(path: Path) -> dict:
     """Return diskutil plist info for a path on macOS, or empty dict if unavailable."""
     if platform.system() != "Darwin":
         return {}
     try:
         proc = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)